src/TextField/index.tsx

```diff
@@ -0,0 +1,144 @@
+import getClassName from 'getclassname';
+import React, { useState } from 'react';
+import { Icons } from '../sharedTypes';
+
+interface TextFieldPropsWithoutRef {
+  /**
+   * id attribute to be passed to the underlying input element. Will default to `name`, then `label` props
```

Comment:
Minor detail, try to be consistent with the comments case, b/c some are capitalized and others are in lowercase. 🙏🏾 This convention is also reflected in the `README.md` file

Reply:
Sure thing. Also I made it more consistent with formatting